content/docs/command-reference/spec.json

```diff
@@ -0,0 +1,734 @@
+      },
+      {
+        "decl": "--absolute, --abs / --relative, --rel",
+        "help": "Which path to linked object to specify: absolute or relative. [default: relative]"
+      },
+      {
+        "decl": "--help",
+        "help": "Show this message and exit."
+      }
+    ],
+    "usage": "Usage: mlem link [options] source target",
+    "doc": "Create a link (read alias) for an existing MLEM Object, including from\nremote MLEM projects."
+  },
+  "types": {
+    "args": {
+      "args": [
+        {
+          "decl": "[ABC]",
+          "help": "Subtype to list implementations. List subtypes if not provided"
+        },
+        {
+          "decl": "[SUB_TYPE]",
+          "help": "Type of `meta` subtype"
+        }
+      ],
+      "impls": null,
+      "impl_metavar": null,
+      "subcommands": null
+    },
+    "options": [
+      {
+        "decl": "--help",
+        "help": "Show this message and exit."
+      }
+    ],
+    "usage": "Usage: mlem types [options] [abc] [sub_type]",
+    "doc": "List different implementations available for a particular MLEM type. If a\nsubtype is not provided, list all available MLEM types."
+  },
+  "apply-remote": {
+    "args": {
+      "args": [
+        {
+          "decl": "DATA",
+          "help": "Path to data object [required]"
+        }
+      ],
+      "impls": [
+        "http",
+        "rmq"
+      ],
+      "impl_metavar": "client",
+      "subcommands": null
+    },
+    "options": [
+      {
+        "decl": "-p, --project TEXT",
+        "help": "Path to MLEM project [default: (none)]"
+      },
+      {
+        "decl": "--rev TEXT",
+        "help": "Repo revision to use [default: (none)]"
+      },
+      {
+        "decl": "-o, --output TEXT",
+        "help": "Where to store the outputs."
+      },
+      {
+        "decl": "--target-project, --tp TEXT",
+        "help": "Project to save target to [default: (none)]"
+      },
+      {
+        "decl": "-m, --method TEXT",
+        "help": "Which model method is to be applied [default: predict]"
+      },
+      {
+        "decl": "--index / --no-index",
+        "help": "Whether to index output in .mlem directory"
+      },
+      {
+        "decl": "--json",
+        "help": "Output as json"
+      },
+      {
+        "decl": "-l, --load TEXT",
+        "help": "File to load client config from"
+      },
+      {
+        "decl": "-f, --file_conf TEXT",
+        "help": "File with options for client in format `field.name=path_to_config`"
+      },
+      {
+        "decl": "--help",
+        "help": "Show this message and exit."
+      }
+    ],
+    "usage": "Usage: mlem apply-remote client [options] data",
+    "doc": "Apply a deployed-model (possibly remotely) to data. The results will be saved as\na MLEM object to `output` if provided. Otherwise, it will be printed to\n`stdout`."
+  },
+  "build": {
+    "args": {
+      "args": [
+        {
+          "decl": "MODEL",
```

Comment:
Does the order in this file matter? This should be the 2nd entry if we go alphabetically.

Reply:
I guess not: it's just the order in which .md files will be processed